blocks/components/editable/index.js

```diff
@@ -54,7 +54,9 @@ export default class Editable extends wp.element.Component {
 		}
 		const value = this.editor.getContent();
 		this.editor.save();
-		this.props.onChange( value );
+		// We need the node itself to parse the content and being
+		// able to correctly swap tags.
+		this.props.onChange( value, this.node );
```

Comment:
I agree. I think that if we are regularly going to be trying to convert a model (e.g. items) into an HTML string for input into the editor, then unless we always want to have to parse that HTML string when updating the attributes (and potentially any root node of the editor), `value` may not be the most generically useful field. A named object has the advantage of being relatively future-proof, but can add bloat to the code.

I think in this situation, we were still entertaining the notion that an update may change the actual root node in some way, which is probably not necessary. It might make more sense for actions that are changing the root node to be performed through `controls` here, rather than in the editor itself.